app_utils/chat.py

```diff
@@ -9,7 +9,7 @@
 API_ENDPOINT = "https://{HOST}/api/v2/cortex/analyst/message"
 
 
-@st.cache_data(ttl=60, show_spinner=False)
+@st.cache_data(ttl=300, show_spinner=False)
```

Comment:
why do we need update it to 300?

Reply:
We don't need to, I just thought this was short and bumped it for my own debugging. If we feel like we shouldn't change, happy to have it reverted.

Reply:
will revert for something smaller.